backend/routes/quests.go

```diff
@@ -103,3 +118,100 @@ func GetTodayStartTime(w http.ResponseWriter, r *http.Request) {
 
 	WriteDataJson(w, "\""+string(todayStartTime.UTC().Format(time.RFC3339))+"\"")
 }
+
+func SendChat(w http.ResponseWriter, r *http.Request) {
+	userAddress := r.URL.Query().Get("address")
+	if userAddress == "" {
+		WriteErrorJson(w, http.StatusBadRequest, "Missing address parameter")
+		return
+	}
+
+	var chat FactionChat
+	err := json.NewDecoder(r.Body).Decode(&chat)
+	if err != nil {
+		WriteErrorJson(w, http.StatusBadRequest, "Invalid request body")
+		return
+	}
+	defer r.Body.Close()
+
+	err = core.PostgresExec(r.Context(), "INSERT INTO FactionChats (sender, faction_key, message) VALUES ($1, $2, $3)", chat.Sender, chat.FactionKey, chat.Message)
```

Comment:
The sender can be set to a hardcoded address for now ( 0 is fine ), also only insert into factionchats if the sender is in the corresponding faction.

Please do all postgres queries / execs in a single request as well to prevent back and forth with the postgres server. There are various ways to do conditional inserts within a single query with sql